vignettes/04-synthesis-data.Rmd

```diff
@@ -250,12 +246,12 @@ We then determine the average canopy cover across the site for the day that the
 The relationship between our greenness metric and average canopy cover are plotted. 
 
 ```{r plot_sensor_trait}
-trait_canopy_cover <- trait_canopy_cover %>% 
+trait_canopy_cover3 <- trait_canopy_cover2 %>% 
```

Comment:
Maybe use traits_canopy_cover_and_greeness or traits_merged? I think it'd be clearer

Reply:
daily_canopy_cover?